lnbits.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Check install has not already run
+if [ ! -d lnbits/data ]; then
+
+  # Update package list and install prerequisites
+  sudo apt update
+  sudo apt install -y software-properties-common curl git
+  sudo add-apt-repository -y ppa:deadsnakes/ppa
+  sudo apt install -y python3.9 python3.9-distutils
+
+  # Install Poetry
+  curl -sSL https://install.python-poetry.org | python3 -
+
+  # Add Poetry to PATH for the current session
+  export PATH="$HOME/.local/bin:$PATH"
+
+  if [ -d lnbits ]; then # Someone didnt follow the install guide
+    # Check out the main branch
+    git checkout main
+  else
+    # Clone the LNbits repository
+    git clone https://github.com/lnbits/lnbits.git
+    cd lnbits
+
+    # Check out the main branch
+    git checkout main
```

Comment:
`git checkout main` runs on both branches of the `if`. Is the `if` required?

Reply:
fixed